.github/workflows/main.yml

```diff
@@ -55,6 +55,12 @@ jobs:
           wget ${BASE_URL}/MacOS/sac2c-basic.pkg
           sudo installer -pkg ./sac2c-basic.pkg -target /
           sac2c -V
+      - name: Set XCode version
+        run: |
+          # this ensures we are using the same Xcode version that we compiled sac2c against
+          sudo xcode-select -switch "/Applications/Xcode_12.app"
+          # we need to rewrite sysroot path to use specific Xcode version
+          sudo find /usr/local/share -type f -name 'sac2crc_*' -execdir sed -i.bak 's/Xcode\.app/Xcode_12\.app/g' {} \;
```

Comment:
This is a kludge, I know. Ideally we need to change how CMake in the sac2c project determines the SDK path. Seemingly Xcode encodes a generic (version free) path, but CMake clearly doesn't use this. For the moment this solution works.